sources/packages/backend/workflow/test/2022-2023/parttime-assessment/appeals/parttime-assessment-partner-information-and-income.e2e-spec.ts

```diff
@@ -124,4 +124,30 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-partner-informat
       calculatedAssessment.variables.calculatedDataPartner1TotalIncome,
     ).toBeNull();
   });
+
+  it("Should calculate partner income as current year partner income value when there is a request a change for current year partner income.", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.partner1CRAReportedIncome = 3000;
```

Comment:
Please add one more test to ensure the precedence against , `calculatedDataPartner1TotalIncome` which can be taken from `partner1TotalIncome` when `partner1TotalIncome` is not null.

![image](https://github.com/user-attachments/assets/53202098-49e1-4df1-821c-3c85bf0fd644)